### PR TITLE
Added missing framework references for WinUI

### PR DESCRIPTION
**Pull request by Haplois**
* Added missing framework references
 * Updated license headers on PowerShell files
 * Included a script to install required Windows SDK

### CODE REVIEW

File: scripts/Install-WindowsSDK.ps1 [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    $currentColor = $Host.UI.RawUI.ForegroundColor
+    $Host.UI.RawUI.ForegroundColor = $messageColor
+    if ($message) {
+      Write-Output "... $message"
+    }
+    $Host.UI.RawUI.ForegroundColor = $currentColor
+}
+
+Push-Location
+try {
```
Comment (nohwnd) [inline scripts/Install-WindowsSDK.ps1]: Why do you do that? To stay in the same folder afterwards? What can change it?

File: scripts/Install-WindowsSDK.ps1 [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+function Write-Log ([string] $message, $messageColor = "Green") {
+    $currentColor = $Host.UI.RawUI.ForegroundColor
```
Comment (nohwnd) [inline scripts/Install-WindowsSDK.ps1]: Why is this not using Write-Host? Also RawUI can be null.

**Review by nohwnd — CHANGES_REQUESTED** [review]

File: scripts/Install-WindowsSDK.ps1 (hunk 1 above)
Comment (Haplois) [reply scripts/Install-WindowsSDK.ps1]: Haha - Copy paste artifact. (I normally change it to temp folder and download there, I'll use this script in pipelines so removed that functionality. Fixing...)

File: scripts/Install-WindowsSDK.ps1 (hunk 2 above)
Comment (Haplois) [reply scripts/Install-WindowsSDK.ps1]: Grabbed that from our build script, no idea. Simplifying..

File: scripts/Install-WindowsSDK.ps1 [hunk 3]
```diff
@@ -1,34 +1,18 @@
...
-      Write-Output "... $message"
-    }
-    $Host.UI.RawUI.ForegroundColor = $currentColor
-}
-
-Push-Location
-try {
-    Write-Log "Downloading the Windows SDK 10.0.14393.795..."
-    Invoke-WebRequest -Method Get -Uri https://go.microsoft.com/fwlink/p/?LinkId=838916 -OutFile sdksetup.exe -UseBasicParsing
+Write-Host "Downloading Windows SDK 10.0.14393.795..." -ForegroundColor "Green"
```
Comment (nohwnd) [inline scripts/Install-WindowsSDK.ps1]: nit: quotes around Green are not needed.

**Review by nohwnd — APPROVED** [review]
Your build is failing though.

**Comment by Haplois** [comment]
> Your build is failing though.

That's fixed too :)

**Pull request merged**